schema/campaign_spec.schema.json

```
@@ -56,6 +56,28 @@
         ]
       }
     },
+    "workspaces": {
+      "type": "array",
+      "description": "Different workspace configurations",
```

```suggestion
      "description": "Individual workspace configurations",
```

maybe? I didn't really understand what is _different_ here

---

Yes, correct. Sorry, didn't put a lot of effort into the descriptions this time.